MediaRecorder.html

```diff
@@ -326,11 +329,11 @@
             playable, but the combination of all the Blobs from a completed
             recording MUST be playable.</p>
 
-            <p> If any Track within the MediaStream is muted at any time (i.e.,
-            if its <code>muted</code> flag is set to <code>true</code>), the UA
-            MUST insert black frames or silence until the Track is unmuted, or
-            record nothing, depending on the value of
-            <a>ignoreMutedMedia</a>.</p>
+            <p> The UA MUST insert black frames or silence, or record nothing,
+            depending on the value of <a>ignoreMutedMedia</a>, for as long as
+            a given <var>track</var> within the MediaStream  is
+            <a href="https://www.w3.org/TR/mediacapture-streams/#widl-MediaStreamTrack-muted">
+            muted</a>. This behaviour only applies to the muted tracks.</p>
 
```

Comment:
This still does not address the question of what happens if you do

getUserMedia(audio: true, video:true).then(stream => stream.getVideoTracks()[0].muted = true)

and record the result.

Suggested rephrase:

If all tracks are muted, and the "ignoreMutedMedia" flag is set to "true", record nothing (stop adding data to the recording).
Otherwise, the UA MUST insert black frames or silence until the track is unmuted.